_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
just having the vjs-fluid class makes it work nicely once it's loaded but until it's loaded it looks pretty bad, so, we force the height correctly.